### ISSUE TITLE
Missing Input_Line function when -E option is used

### ISSUE THREAD

**Issue by stcarrez** [Ada-France/aflex#2]
When the `-E` option is used with aflex, we are supposed to generate a `Input_Line` function for AYacc.
I've removed that function in Aflex 1.5 due to a miss-understanding on my side.

With the new template implementation and its `%if/%else/%end` condition, it's now easier to resurrect the old `Input_Line` function.

When `-E` is not given, the `Input_Line` function returned only 1 and was not necessary.

See https://github.com/Ada-France/ayacc/issues/3

**Comment by zertovitch** [comment]
Tremendous!
I have tested on p2ada and the only glitch so far is a bunch of "Text_IO"'s that have to become "Ada.Text_IO"'s.
The reason is that some `with` clauses are now "with Ada.Text_IO" instead of "with Text_IO".
Then, GNAT complains with a little bit odd message:
```
pascal_io.ads:57:34: "Text_IO" is not visible
pascal_io.ads:57:34: non-visible declaration at a-textio.ads:58
```
Patch attached: [aflex.patch](https://github.com/Ada-France/aflex/files/10429424/aflex.patch).
The file `template_manager-templates.ads` needs to be regenerated properly (for testing, I have added the "Ada."'s by hand in that file).
I have also added a line for the `.gpr` file to make GNAT Studio users happy (with that line, the build works "out of the box").
